logzio-sender/src/test/java/io/logz/sender/LogzioLongRunningTests.java

```diff
@@ -16,38 +17,39 @@
 import java.util.List;
 import java.util.concurrent.CountDownLatch;
 import java.util.concurrent.Executors;
+import java.util.concurrent.ScheduledExecutorService;
 import java.util.concurrent.TimeUnit;
-import static org.assertj.core.api.Assertions.assertThat;
-import static io.logz.sender.LogzioTestSenderUtil.createJsonMessage;
 
-/**
- * @author MarinaRazumovsky
- */
+import static io.logz.sender.LogzioTestSenderUtil.createJsonMessage;
+import static org.assertj.core.api.Assertions.assertThat;
 
 public class LogzioLongRunningTests {
-
     private final static Logger logger = LoggerFactory.getLogger(LogzioLongRunningTests.class);
     private MockLogzioBulkListener mockListener;
+    private ScheduledExecutorService tasks;
 
     @Before
-    public void startMockListener() throws Exception {
+    public void preTest() throws Exception {
```

Comment:
Before and After method should  match by name

Reply:
@asafm Not sure I understand you

Reply:
preTest and postTest
startX, stopX